pygmt/src/surface.py

```diff
@@ -68,6 +74,45 @@ def surface(data=None, x=None, y=None, z=None, **kwargs):
     outgrid : str
         Optional. The file name for the output netcdf file with extension .nc
         to store the grid in.
+    convergence : float
+        Optional. Convergence limit. Iteration is assumed to have converged when the maximum 
+        absolute change in any grid value is less than convergence_limit. 
+        (Units same as data z units). Alternatively, give limit in percentage 
+        of rms deviation by appending %. [Default is scaled to  of the 
```

Comment:
This is how you can add $10^{-4}$ to the docstring.
```suggestion
        of rms deviation by appending %. [Default is scaled to :math:`10^{{-4}}` of the 
```